src/rpc/client/ts/scripts/package.ts

```diff
@@ -45,58 +45,30 @@ function copyFiles(srcDir, destDir, extension = '') {
   })
 }
 
-// Setup package directory
-function setup() {
-  if (fs.existsSync(pkg_dir)) {
-    fs.rmSync(pkg_dir, { recursive: true, force: true })
-  }
-
-  // Copy generated files to the package directory
-  copyFiles('src', pkg_dir, '.d.ts')
-  copyFiles('src', pkg_dir, '.js')
-  copyFiles('out', pkg_dir)
-
+// Create package
+function package() {
   // Copy the server directory to the package directory
   copyFiles(
     `omega-edit-grpc-server-${pkg_version}`,
     path.join(pkg_dir, `omega-edit-grpc-server-${pkg_version}`)
   )
```

Comment:
The webpack copy plugin can do this (see: https://github.com/webpack-contrib/copy-webpack-plugin).  I prefer the flatter layout of just having the `bin/` and `lib/` directories though.